### ISSUE TITLE
App Crashes

### ISSUE THREAD

**Issue by McIntyre227** [Swartdogs/PowerScout#66]
When I try to open the app, it crashes. On Friday, Matthew loaded the latest version of the app on the 2 iPads that 525 owns. We entered mock data to test the app function. Everything worked very well, except the Bluetooth. Today (Sunday) when I tried to open the app, it crashed and would not open, very much like what happened to us at Cow Town last October. After I pressed the PowerUp Scout icon, the opening screen came up for about 1 second and then disappeared back to the home screen of the iPad. I get the same result every time I try to open the app. I have restarted the iPad, but the problem remains. Since I do not have macbook, I am not able to reinstall the app and I am handing this issue off to one of you guys.

**Comment by Pr1sM** [comment]
This definitely sounds related to #65. If you connect to the iTunes File Sharing and delete any .archive files, is the app able to load?

**Comment by Pr1sM** [comment]
I'm wondering if the data transfer corrupted some of the data in the archive files. These files get loaded when the app starts and if there is corrupted data in them, that would cause the app to crash. 

You also mentioned 2 devices -- are both apps crashing, or is it just one device?

**Comment by McIntyre227** [comment]
I don't have the iPads with me. I will need to wait until late tomorrow afternoon before I can delete any of the .archive files and see if that makes a difference. I will try this to see if it works. Both of the iPads I was using crashed, so yes, it's happening to both of them.

**Comment by McIntyre227** [comment]
I have deleted the .archive files on both iPads and the app still does not load.

**Comment by Pr1sM** [comment]
Would you be able to send me the .csv files and the .archive files from both devices? @MattDaoud225 can load the app again connected to Xcode so he can see the console output. When the app crashes, there should be a error message the appears on the console. 

Another thing to check is the Model number of for the iPads. On the back of the iPad there should be a Model number printed below the Apple logo. If that's covered, you should also be able to see the Model number in the settings by going to Settings > General > About.

**Comment by McIntyre227** [comment]
The iPads are model #MD339LL/A 16GB. I had deleted the .archive files so I can't send them to you. Here are the last .csv file from both iPads. When I went to attach the .csv file it said it didn't support this file type, but did support .xlsx so I saved the .csv as .xlsx files. They are attached.

[Match data - FRC 525 iPad#2.xlsx](https://github.com/Swartdogs/PowerScout/files/1760842/Match.data.-.FRC.525.iPad.2.xlsx)


[Match data - FRC 525 Ipad#1.xlsx](https://github.com/Swartdogs/PowerScout/files/1760825/Match.data.-.FRC.525.Ipad.1.xlsx)

**Comment by Pr1sM** [comment]
👍 I'll try recreating the data on my devices and see if the same crash occurs.

**Comment by Pr1sM** [comment]
I believe we've concluded the cause of this is because of the 1 week expirations of free signing certificates. I've signed the app with my developer certificate and have shared the iOS distribution profile and needs to be installed on devices in order to install the app. I then archived the app and uploaded it to Slack so a stable version of the app can be installed by anyone. 

For normal testing this is probably not necessary, but depending how the app runs during this weekend's competition, we might want to do this for competition dates.
